commons/utils/job-store-service-connector/src/test/java/dk/dbc/dataio/commons/utils/jobstore/JobStoreServiceConnectorTest.java

```diff
@@ -59,8 +59,7 @@ public class JobStoreServiceConnectorTest {
 
     private final HttpClient httpClient = mock(HttpClient.class);
 
-    private final JobStoreServiceConnector jobStoreServiceConnector =
-            new JobStoreServiceConnector(httpClient, JOB_STORE_URL);
+    private final JobStoreServiceConnector jobStoreServiceConnector = new JobStoreServiceConnector(httpClient, JOB_STORE_URL, null);
```

Comment:
Kunne man ikke overloade constructorne i stedet for? Det ser en smule mærkeligt ud at de alle skal instantieres med en null værdi :)

Reply:
Mjoh